src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -117,7 +117,21 @@ private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
    * @throws CloudSdkOutOfDateException when the installed Cloud SDK is too old 
    */
   public void runAppCommand(List<String> args) throws ProcessRunnerException {
-    runGcloudCommand(args, "app");
+    runGcloudCommand(args, null, "app");
```

Comment:
The problem is that in code reviews, we can't see that IJ auto completion :P
However you prefer, though.